internal/services/portal/portal_dashboard_data_source_test.go

```diff
@@ -73,3 +86,21 @@ data "azurerm_portal_dashboard" "test" {
 }
 `, PortalDashboardResource{}.complete(data), resourceName)
 }
+
+func (PortalDashboardDataSource) displayName(data acceptance.TestData) string {
+	resourceName := "azurerm_portal_dashboard"
+	if !features.ThreePointOhBeta() {
+		resourceName = "azurerm_dashboard"
+	}
```

Comment:
we're past the 3.0 release here:

```suggestion
	resourceName := "azurerm_portal_dashboard"
```

Reply:
Updated